dat/scripts/luatk/core.lua

```diff
@@ -363,12 +363,13 @@ function luatk.Window:draw()
 
    -- Draw background
    lg.setColour( luatk.colour.bg )
-   lg.rectangle( "fill", x, y, w, h )
```

Comment:
Would probably make sense to have the 30s be some sort of variable instead of hardcoded all over.